pom.xml

```diff
@@ -0,0 +1,184 @@
+    <artifactId>arpnetworking-parent-pom</artifactId>
+    <version>1.0.29</version>
+    <relativePath />
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.arpnetworking.metrics.extras</groupId>
+  <artifactId>incubator-extra</artifactId>
+  <packaging>jar</packaging>
+  <name>Metrics client incubator</name>
+  <description>Incubator for features that will later be built into the core metrics client.</description>
+  <url>https://github.com/ArpNetworking/metrics-incubator-extra</url>
+  <version>0.6.0-SNAPSHOT</version>
+
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>brandonarp</id>
+      <email>brandon.arp@inscopemetrics.com</email>
+      <organization>Inscope Metrics</organization>
+      <organizationUrl>http://www.inscopemetrics.com</organizationUrl>
+      <roles>
+        <role>developer</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>villekoskela</id>
+      <email>ville.koskela@inscopemetrics.com</email>
+      <organization>Inscope Metrics</organization>
+      <organizationUrl>http://www.inscopemetrics.com</organizationUrl>
+      <roles>
+        <role>developer</role>
+      </roles>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:arpnetworking/metrics-incubator-extra.git</connection>
+    <developerConnection>scm:git:git@github.com:arpnetworking/metrics-incubator-extra.git</developerConnection>
+    <url>https://github.com/arpnetworking/metrics-incubator-extra</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <!-- IMPORTANT: Any changes to dependencies, including versions, require
+      validation that the shading rules correctly resolve the transitive closure
+      of all non-test packages. You can validate that the transitive closure of
+      dependencies was covered by checking the pom file generated by the shade
+      plugin: dependency-reduced-pom.xml.
+
+      In general, there are two strategies for dealing with additional
+      dependencies:
+
+      1) If we depend on package A and it on package B but we don't use any
+      features that require package B we may exclude those classes from package
+      A and prune the dependency on package B. This will remove package B.
+
+      2) If we depend on package A and it on package B and we cannot prune the
+      dependency then include the package and classpath(s) for package B in the
+      shade plugin's mapping. This will hide package B.
+    -->
+
+    <!-- Dependency versions -->
+    <slf4j.version>1.7.22</slf4j.version>
+    <metrics.client.version>0.6.0</metrics.client.version>
+    <arpnetworking.commons.version>1.10.0</arpnetworking.commons.version>
+    <jsr305.version>3.0.1</jsr305.version>
+    <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
+
```

Comment:
I don't think we've separated test and compile in this list (single, alpha-order list).